content/CORE/CORETutorials/Services/ConfiguringFTP.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Configuring FTP"
+description: "Configuring FTP on your TrueNAS"
```

Comment:
The description parameter is a little too close to the title, so maybe make it more of a complete sentence? "This article provides information on how to configure File Transfer Protocol (FTP) on your TrueNAS." or some variation. Can be "This article provides instructions on to to set up FTP using **Services > FTP**. There are a couple of different ways to be more descriptive but not to verbose in the short title.